src/drive/web/modules/filelist/File.jsx

```diff
@@ -62,10 +62,26 @@ const File = props => {
   }
 
   const open = (event, attributes) => {
-    const { onFolderOpen, onFileOpen, isAvailableOffline } = props
+    console.log('open')
+
+    const {
+      onFolderOpen,
+      onFileOpen,
+      isAvailableOffline,
+      folderUrlToNavigate
+    } = props
     event.stopPropagation()
     if (isDirectory(attributes)) {
-      onFolderOpen(attributes.id)
+      console.log('is directory')
+
+      if (event.ctrlKey || event.metaKey || event.shiftKey) {
```

Comment:
it seems that you build an href for a folder https://github.com/cozy/cozy-drive/pull/2521/files#diff-66f571a15fdeccd6293e111e9483d731a1d8a1bddf527123ece478a9605e12bfR121 why do you need to check the event key?